packages/core/src/internal-components/popup/popup.element.ts

```diff
@@ -101,8 +102,7 @@ export class CdsInternalPopup extends CdsInternalStaticOverlay implements Animat
    * do not manipulate the responsive property because the
    * positioning logic may override your changes.
    */
-  /** @private */
-  @property({ type: Boolean, reflect: true })
+  @state({ type: Boolean, reflect: true, attribute: 'responsive' })
```

Comment:
Fix for lit code linter to use `@state` on private api properties rather than `@property`

Reply:
🤔

I made this a "private" property on purpose because code outside of the popup (like the animations) needed to reference it. If we are reflecting it, it may _still_ work because I added smarts to the animations to look up values in attributes (and not just object properties).

We just want to be sure that the responsive modals/dropdowns and animations work as expected here.

Reply:
It still works, `@state` uses `@property` under the hood defaulting to not reflecting. It's just a way to mark that it's a private/internal property rather than public to consumers outside of using TS. We do this in a lot of place where we need to communicate between components but mark it as a non-public API. The web component metadata generator will treat `@state` as private https://lit.dev/docs/api/decorators/#state I went ahead and added the jsdoc private back